users/vieting/experiments/switchboard/hybrid/feat/experiments.py

```diff
@@ -208,8 +208,32 @@ def run_baseline_scf():
         # noinspection PyUnresolvedReferences
         train_job.rqmt.update({"gpu_mem": 24, "mem": 10})
 
+def run_specaug_scf():
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/hybrid/feat/"
+
+    nn_args = get_nn_args_baseline(
+        nn_base_args={
+            "scf": dict(
+                returnn_args=dict(batch_size=1000),
+                feature_args={"class": "ScfNetwork", "size_tf": 256 // 2, "stride_tf": 10 // 2}
+            )
+        },
+        prefix="conformer_bs14k_",
```

Comment:
It would be better to introduce a different prefix (or at least a different name for the experiment). Right now, this experiment will have the same alias as the experiment in `run_baseline_scf`.